MonoGame.Framework.Content.Pipeline/ContentImporter.cs

```diff
@@ -31,6 +32,31 @@ protected ContentImporter()
         public abstract T Import(string filename, ContentImporterContext context);
 
         /// <summary>
+        /// Called by external third-party systems when importing a game asset.  This method is used when the asset is being imported from a stream.
+        /// </summary>
+        /// <param name="input">The stream to read the asset from.</param>
+        /// <param name="virtualFilename">For importers that depend on filenames (such as file extensions), this provides a virtual filename for those importers.</param>
+        /// <param name="context">Contains information for importing a game asset, such as a logger interface.</param>
+        /// <returns>Resulting game asset.</returns>
+        public virtual T Import(Stream input, string virtualFilename, ContentImporterContext context)
+        {
+            var temp = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName() + Path.GetExtension(virtualFilename));
+            try
+            {
+                using (var writer = new FileStream(temp, FileMode.Truncate, FileAccess.Write))
```

Comment:
@KonajuGames @dellis1972 

I think I would rather see us break compatibility with XNA 4 here rather than have a method like this.

Or maybe implement a second class like `ContentImporter2` or whatever that only has the stream interface for opt-in to the new streaming feature?